Tests/MapboxMapsTests/Foundation/Extensions/UIWindow+ParentSceneTests.swift

```diff
@@ -0,0 +1,20 @@
+import XCTest
+import CarPlay
+@testable import MapboxMaps
+
+final class UIWindowParentSizeTests: XCTestCase {
+
+    @available(iOS 13.0, *)
+    func testCarPlayWindowReturnsCorrectParentScene() {
+        let window = CPWindow()
+
+        XCTAssertEqual(window.parentScene, window.templateApplicationScene)
```

Comment:
What are the actual values? Are they both nil? If so, that might not be a very good test.

Reply:
Not for this PR, but I wonder whether we could set up a test suite that runs in a host app with CarPlay entitlements.

Reply:
Good point, this will enable for tests like this to be run properly, let's get back to it next time we touch CarPlay related stuff.